src/app/pages/news/9.3/02-config-files.md

```diff
@@ -1,19 +1,8 @@
 ## Config files {#project93}
 
-... SCM ready, easy DevOps, new tree, ...
-
-<div class="short-links">
-	<a href="${docBaseUrl}/designer-guide/configuration/rest-clients.html#openapi-client-generator" target="_blank" rel="noopener noreferrer">
-		<i class="si si-check"></i> OpenAPI client
-	</a>
-	<a href="${docBaseUrl}/designer-guide/process-modeling/process-elements/rest-client-activity.html#process-element-rest-client-activity-browser" target="_blank" rel="noopener noreferrer">
-		<i class="si si-check"></i> API browser
-	</a>
-	<a href="${docBaseUrl}/designer-guide/configuration/rest-clients.html#rest-clients-openapi-migrate" target="_blank" rel="noopener noreferrer">
-		<i class="si si-check"></i> OpenAPI converters
-	</a>
-	<a href="${docBaseUrl}/public-api/ch/ivyteam/ivy/rest/client/oauth2/OAuth2BearerFilter.html" target="_blank" rel="noopener noreferrer">
-		<i class="si si-movie"></i> Public API: OAuth2
-	</a>
-</div>
+The entire configuration in Axon Ivy projects has been revised and consolidated.
+This will make the daily life for all developers much easier.
 
+- __Streamlining__ : The configuration of rest clients, web service clients, databases or variables now looks exactly the same in your Axon Ivy project as on the Axon Ivy Engine.
```

Comment:
another idea: ready to customize by copying into app.yaml ... make these configs sexy not only for developers, but operations too